### ISSUE TITLE
Editor isn't accessible to screen readers

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-hexeditor#29]
Some problems:

- The main document cannot be focused via tab
- Tab doesn't work when the main document has focus and arrows don't work either

**Comment by lramos15** [comment]
Agreed. Need to look into this, see #4
